pkg/controller/apiextensions/composite/reconciler.go

```diff
@@ -93,8 +100,8 @@ type Resolver interface {
 	ResolveSelector(ctx context.Context, cr resource.Composite) error
 }
 
-// compositeReconciler reconciles the generic CRD that is generated via InfrastructureDefinition.
-type compositeReconciler struct {
+// Reconciler reconciles the generic CRD that is generated via InfrastructureDefinition.
+type Reconciler struct {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#interfaces
https://mycodesmells.com/post/accept-interfaces-return-struct-in-go

This was mostly a reflexive nitpick refactoring, but I think it's a good practice to "accept interfaces and return concrete types" in Go. The current implementation was returning an interface because the concrete type was not public, so the change was more about not returning an interface than it was about making the type public.